docs/guide/date-and-time-handling.md

```diff
@@ -24,40 +17,31 @@ const options = {
 };
 ```
 
-## Integration and customization
+## Custom functions for handling date and time 
 
 HyperFormula offers the possibility to extend the number of supported
 date/time formats as well as the behavior of this functionality by exposing
 three options:
 
-* `parseDateTime` which allows for providing a function that accepts
+- [parseDateTime](../api/interfaces/configparams.md#parsedatetime), which allows to provide a function that accepts
 a string representing date/time and parses it into an actual date/time format
-* `stringifyDateTime`which allows for providing a function that
+- [stringifyDateTime](../api/interfaces/configparams.md#stringifydatetime), which allows to provide a function that
 takes the date/time and prints it as a string
-* `stringifyDuration` which allows for providing a function that
+- [stringifyDuration](../api/interfaces/configparams.md#stringifyduration), which allows to provide a function that
 takes time duration and prints it as a string
 
-To extend the number of possible date formats you will need to
-configure `parseDateTime` . This functionality is based on callbacks
+To extend the number of possible date formats, you will need to
+configure `parseDateTime` . This functionality is based on callbacks,
 and you can customize the formats by integrating a third-party
 library like [Moment.js](https://momentjs.com/), or by writing your
-own custom function like this:
-
-```javascript
- {
-  year: number,
-  month: number,
-  day: number,
- }
-```
```

Comment:
Can you add a fuller example that is actually a function? This snippet is not clear.

Reply:
There is a full example using Moment.js in the next section. Do you think, we need another example?

Reply:
Would be good, but the link to `/api/interfaces/globals.html#datetime` gives a 404 to me

Reply:
Sorry about that. It's fixed now.